src/erl.rs

```diff
@@ -1380,3 +1486,163 @@ pub fn is_erlang_standard_library_module(name: &str) -> bool {
             | "zip"
     )
 }
+
+impl TypeAst {
+    pub fn to_erlang_type_def(&self) -> Document {
+        // reference : https://erlang.org/doc/reference_manual/typespec.html
+        match self {
+            TypeAst::Var { name, .. } | TypeAst::Hole { name, .. } => variable_name(name).to_doc(),
+            TypeAst::Tuple { elems, .. } => "{"
```

Comment:
I think there is a `tuple` function that can be used here. It'll handle wrapping and indentation and such.